spec/features/user_deletes_attachment_from_a_coverage_spec.rb

```diff
@@ -0,0 +1,31 @@
+require "rails_helper"
+
+feature "User views attachment count and deletes an attachment from
+  a coverage", js: true do
+  scenario "successfully" do
+    coverage = create(:coverage)
+    attachment = create(:attachment, attachable: coverage)
+    attachment_count = coverage.attachments.size
+    subject_name = coverage.subject.title
+    user = user_with_assignments_access_to(coverage.course.department)
+
+    visit manage_assignments_course_path(coverage.course.id, as: user)
+
+    within(".class-card-assignment-control") do
+      expect(page).to have_content(attachment_count)
+    end
+
+    click_on t("manage_assignments.outcome_coverages.outcome_coverage.attachments-expandable-link")
+
+    within(".assignment-attachments") do
+      click_on t("manage_assignments.shared.attachments.delete_link")
+    end
```

Comment:
Is there a better way to describe what the interaction is here? I know that we're clicking on a link/button within a certain part of the page, but it's not entirely clear as to what's going on from a glance.

I'm thinking something like the following:

```ruby
scenario "..." do
  # ...
  delete_assignment
end

def delete_assignment
  within ... do
    click_on ...
  end
end
```